### ISSUE TITLE
Provide default value of term as Dart doc on getters in l10n.dart

### ISSUE THREAD

**Issue by orestesgaolin** [localizely/flutter-intl-vscode#29]
I would be very happy to see Dart doc comments over getters in l10n.dart file with default translation. 

For instance right now it's:

```dart
  String get common_next {
    return Intl.message(
      'Dalej',
      name: 'common_next',
      desc: '',
      args: [],
    );
  }
```

And it would be awesome to have:

```dart
  /// `Dalej`
  String get common_next {
    return Intl.message(
      'Dalej',
      name: 'common_next',
      desc: '',
      args: [],
    );
  }
```

This way I could select proper term without peeking into the file.

**Comment by aleksakrstic** [comment]
@orestesgaolin 
This is now available in version `1.9.0`

**Comment by orestesgaolin** [comment]
Thanks, it's working great! 
![image](https://user-images.githubusercontent.com/16854239/84049294-8b043800-a9ac-11ea-9522-d5d59d1feaf6.png)
